### PR TITLE
SOS: mca, cfg, sample file, SR functions updates

### PR DESCRIPTION
**Pull request by cbotta**

### CODE REVIEW

File: TTHAnalysis/python/plotter/functions.cc [hunk 1]
```diff
@@ -175,66 +175,76 @@ float mass_tautau( float Met_Pt, float Met_Phi,  float l1_Pt, float l1_Eta, floa
...
-//     else if(met_pt>200.&&20.<Mll&&Mll<30.) return 6.5;
-//     else if(met_pt>200.&&30.<Mll&&Mll<50.) return 7.5;
-//     else return -0.5;
-// }
-
-int SR_bins_EWKino_mm(float Mll, float met_pt, Int_t pdgId_1, Int_t pdgId_2){
+// Plotting SR SOS
+
+int SR_bins_EWKino_100(float Mll, float met_pt, Int_t pdgId_1, Int_t pdgId_2){
   // mm
```
Comment (rcastello) [inline TTHAnalysis/python/plotter/functions.cc]: @cbotta  just for consistency I would suggest to rename this function to  SR_bins_EWKino_100

Comment (rcastello) [inline TTHAnalysis/python/plotter/functions.cc]: Sorry I misread. It's already there.
Thanks

File: TTHAnalysis/cfg/run_susyMultilepton_cfg.py [hunk 2]
```diff
@@ -87,8 +87,8 @@
     lepAna.loose_muon_pt  = 3
     lepAna.inclusive_electron_pt  = 5
     lepAna.loose_electron_pt  = 5
-#    isolation = None
-#    lepAna.loose_electron_id = ""
+    isolation = "absIso03"
```
Comment (peruzzim) [inline TTHAnalysis/cfg/run_susyMultilepton_cfg.py]: This should instead apply the following cut: LepGood_RelIsoFix04*LepGood_pt<10

File: TTHAnalysis/python/plotter/susy-sos/mca-Fall15.txt [hunk 3]
```diff
@@ -3,13 +3,15 @@ TT		: TTJets_DiLepton: xsec: LepGood1_mcMatchId!=0 && LepGood2_mcMatchId!=0; Fil
 
 # ---- Diboson
 WZ 	 : WZTo3LNu: xsec: LepGood1_mcMatchId!=0 && LepGood2_mcMatchId!=0 ;  Label="WZ", FillColor=ROOT.kViolet-4,  NormSystematic=0.3
+WZ 	 : WZTo2L2Q: xsec: LepGood1_mcMatchId!=0 && LepGood2_mcMatchId!=0 ;  Label="WZ", FillColor=ROOT.kViolet-4,  NormSystematic=0.3
 ZZ 	 : ZZTo2L2Nu: xsec: LepGood1_mcMatchId!=0 && LepGood2_mcMatchId!=0  ;  Label="ZZ", FillColor=ROOT.kAzure-9,  NormSystematic=0.3
-WW       : WWTo2L2Nu: xsec: LepGood1_mcMatchId!=0 && LepGood2_mcMatchId!=0  ;  Label="WW", FillColor=ROOT.kAzure-8,  NormSystematic=0.3
```
Comment (peruzzim) [inline TTHAnalysis/python/plotter/susy-sos/mca-Fall15.txt]: Why has WW prompt disappeared?

Comment (cbotta) [inline TTHAnalysis/python/plotter/susy-sos/mca-Fall15.txt]: my bad. putting it back

File: TTHAnalysis/cfg/run_susyMultilepton_cfg.py (hunk 2 above)
Comment (cbotta) [inline TTHAnalysis/cfg/run_susyMultilepton_cfg.py]: aren't we using dR=03 everywhere for the standard PFIso?

File: TTHAnalysis/python/plotter/functions.cc [hunk 4]
```diff
@@ -175,66 +175,76 @@ float mass_tautau( float Met_Pt, float Met_Phi,  float l1_Pt, float l1_Eta, floa
...
-  else if(met_pt>200.&&10.<Mll&&Mll<20.&& pdgId_1*pdgId_2==-121) return 2;
-  else if(met_pt>200.&&20.<Mll&&Mll<30.&& pdgId_1*pdgId_2==-121) return 3;
-  else if(met_pt>200.&&30.<Mll&&Mll<50.&& pdgId_1*pdgId_2==-121) return 4;
+
+int SR_bins_EWKino_200(float Mll, float met_pt, Int_t pdgId_1, Int_t pdgId_2){
+  // mm/ee
+  if     (met_pt>200.&& 4.<Mll&&Mll<=10.&& pdgId_1*pdgId_2==-169) return 1;
+  else if(met_pt>200.&&10.<Mll&&Mll<=20.&& pdgId_1*pdgId_2==-169) return 2;
+  else if(met_pt>200.&&20.<Mll&&Mll<=30.&& pdgId_1*pdgId_2==-169) return 3;
+  else if(met_pt>200.&&30.<Mll&&Mll<=50.&& pdgId_1*pdgId_2==-169) return 4;
```
Comment (rcastello) [inline TTHAnalysis/python/plotter/functions.cc]: @cbotta : this line should be 
else if(met_pt>200.&&30.<Mll&& pdgId_1*pdgId_2==-169) return 4;

**Comment by cbotta** [comment]
now all should be fixed.

File: TTHAnalysis/cfg/run_susyMultilepton_cfg.py (hunk 2 above)
Comment (cbotta) [inline TTHAnalysis/cfg/run_susyMultilepton_cfg.py]: @gpetruc
Marco is using this variable for the lepMVA LepGood_RelIsoFix04 and he is suggesting to use it in the preselection. maybe you want to comment on this for the lepton preselection efficiency. Thanks

**Pull request merged**